pkg/resource/event_bus/sdk.go

```diff
@@ -357,12 +357,12 @@ func (rm *resourceManager) getImmutableFieldChanges(
 	delta *ackcompare.Delta,
 ) []string {
 	var fields []string
-	if delta.DifferentAt("Spec.Name") {
-		fields = append(fields, "Name")
-	}
 	if delta.DifferentAt("Spec.EventBusName") {
 		fields = append(fields, "EventBusName")
 	}
+	if delta.DifferentAt("Spec.Name") {
+		fields = append(fields, "Name")
+	}
```

Comment:
Will be fixed with https://github.com/aws-controllers-k8s/code-generator/pull/405